openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -60,6 +60,7 @@ class LearningContext(TimeStampedModel):
     title = models.CharField(max_length=255)
     published_at = models.DateTimeField(null=False)
     published_version = models.CharField(max_length=255)
+    self_paced = models.BooleanField(default=False)
```

Comment:
Not a full review, but just a quick comment: Please shift this over the `CourseContext` model once #24417 merges.